server/@types/bapv.d.ts

```diff
@@ -140,6 +140,7 @@ export type VisitSlot = {
   endTimestamp: string
   availableTables: number
   visitRoomName: string
+  visitRestriction: string
```

Comment:
Required to know what sort of slot this is for amend journey as this can't be assumed from the availableTables which is based off whichever is the current restriction.